src/stackwalk.c

```diff
@@ -1108,6 +1108,42 @@ JL_DLLEXPORT void jl_print_backtrace(void) JL_NOTSAFEPOINT
     jlbacktrace();
 }
 
+JL_DLLEXPORT void jl_print_task_backtraces(void)
+{
+    jl_task_t *t;
+    jl_ptls_t ptls;
+    arraylist_t *live_tasks;
+    size_t i, j, n;
```

Comment:
nit:
If i'm not mistaken, in the later C standards (C11 onward), which I think we use, you no longer need to declare the variables' types at the top of the function.

I think the preference is (inherited from C++) to keep variable declaration as tightly-scoped as possible, which usually means putting the type declaration on the first assignment.

Does that sound right to you?